origins_unique_title/src/Form/ExcludeTitleForm.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace Drupal\origins_unique_title\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Messenger\MessengerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Implements admin form to exclude nodes ID from the unique title check.
+ */
+class ExcludeTitleForm extends ConfigFormBase {
+
+  /**
+   * Creates a new ExcludeTitleForm instance.
+   */
+  public function __construct(MessengerInterface $messenger) {
+    $this->messenger = $messenger;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container) {
+    return new static(
+      $container->get('messenger')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [
+      'origins_unique_title.excludesettings',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'origins_exclude_title_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('origins_unique_title.excludesettings');
+
+    $message_exclude_ids = "If there are any specific node's ID that shouldn't be validated. List them on new lines";
+
+    $form['exclude_ids_list'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('Excluded Node IDs'),
```

Comment:
Is it OK that the message is being translated here or is it better to translate it where you have it?